cli/packaging/scabbard_circuit_template.yaml

```diff
@@ -0,0 +1,22 @@
+version: v1
+args:
+    - name: $(a:ADMIN_KEYS)
+      required: false
+      default: $(a:SIGNER_PUB_KEY)
+      description: >
```

Comment:
Putting this on its own line causes some weirdness with the way the template is displayed in the `splinter circuit template arguments scabbard` command (there's two blank lines after it instead of just one). Can we just have a long line here to avoid that? Alternatively, could use a different approach for printing out the arguments (though that's more time consuming). Should do the same for this argument in the gameroom template.

Reply:
fixed that by adding `>-` to remove the trailing new line.